src/main/osd/osd_elements.c

```diff
@@ -1451,7 +1451,15 @@ static void osdElementRtcTime(osdElementParms_t *element)
 #ifdef USE_RX_RSSI_DBM
 static void osdElementRssiDbm(osdElementParms_t *element)
 {
-    tfp_sprintf(element->buff, "%c%3d", SYM_RSSI, getRssiDbm());
+    const int8_t antenna = getActiveAntenna();
+    static bool diversity = false;
+
+    if (antenna || diversity) {
+        diversity = true;
+        tfp_sprintf(element->buff, "%d:%c%3d", antenna + 1, SYM_RSSI, getRssiDbm());
```

Comment:
The current crsf code changes the displayed rssidbm between rssi1 & rssi2 based on antenna used.  I still think there is merit in displaying rssi2 in the element e.g. `Ant/RSSI1/RSSI2`, but understand this may not be useful to all.

Reply:
> The current crsf code changes the displayed rssidbm between rssi1 & rssi2 based on antenna used. I still think there is merit in displaying rssi2 in the element e.g. `Ant/RSSI1/RSSI2`, but understand this may not be useful to all.

When I watch 1RSS and 2RSS on my radio, ELRS does not seem to be updating the unused antenna's RSSI dBm. It will remain at the value when it was last used and only the active antenna continues to update. Looking at ELRS code, rx_main.cpp, lines 246-264 seem to have an if/else that updates only one or the other (caveat: always possible I'm missing something in the code)?

If the above is true, showing both RSSI dBm wouldn't add anything...

Reply:
Displaying both will allow you to see either at anytime, and not try to quickly see it when that ant becomes active for a split second.

Reply:
I just hate to extend that element by another 4-5 characters of screen space (assuming some separator, the minus sign, and 2 or 3 digits). That's expensive for the limited value, at least for me.

Reply:
If I can intervene, as I am a bit at the origin of this modification, I agree with @ZzyzxTek . If I need a real follow of my RSSI dBm for each antenna, I can log it in a file into the radio and later make curves and statistics. As I mainly do quiet mid-range flights my idea was to have something easy to read to judge the suitability of the placement of the antennas. Diversity users like me use GPS, and GPS data are using a lot of space in the OSD. Placing a longer line for RSSI dBm in the OSD could be difficult.

@ZzyzxTek , looking at your screenshot, I have a question : doesn't RSSI dBm needs to be [space/minus/2 digits] over -100 dbm to keep a constant size between 2 digits  numbers and 3 digits numbers (not sure I'm clear). Some of my friends use diversity in bando (hopping to have a better average RSSI dBm). The way it is showed could be very "blinky" around -100 with the display of rssi moving left-right and the number of antenna changing 1/2/1/2 ...

But this is only my opinion, you are the bosses

Reply:
> @ZzyzxTek , looking at your screenshot, I have a question : doesn't RSSI dBm needs to be [space/minus/2 digits] over -100 dbm to keep a constant size between 2 digits numbers and 3 digits numbers (not sure I'm clear). Some of my friends use diversity in bando (hopping to have a better average RSSI dBm). The way it is showed could be very "blinky" around -100 with the display of RSSI dBm moving left-right and the number of antenna changing 1/2/1/2 ...

Yeah, I guess the length could jiggle around like that. But tbh, between this being a new feature and the weather preventing me from flying, I haven't had much flight time with it to see if that happens a lot and if it's distracting.